tunnel/tunnel.go

```diff
@@ -257,17 +182,37 @@ func createTunnel(tunnelConfig *Config) (net.Listener, error) {
 	// Listen on remote server port
 	listener, err = sClient.Listen("tcp", remoteEndpoint.String())
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "listen open port ON remote server error: %s", err)
+		fmt.Fprintf(os.Stderr, "listen open port ON remote server error: %s\n", err)
 		return listener, err
 	}
 	log.Debugf("Open listen port on %s", remoteEndpoint.String())
 	return listener, nil
 }
 
+func tunnelStartingSpinner(lock *Semaphore, tunnelStatus *string) {
+	go func() {
+		if !lock.CanRun() {
+			return
+		}
+		defer lock.Done()
+		s := spin.New()
+		for *tunnelStatus == starting {
+			fmt.Printf("\rStarting tunnel %s ", s.Next())
+			time.Sleep(100 * time.Millisecond)
+		}
+		if *tunnelStatus == done {
+			fmt.Printf("\rStarting tunnel ")
+			d := color.New(color.FgGreen, color.Bold)
+			d.Printf("✔\n")
+		}
+	}()
+}
 func cleanup(config *Config) {
+	fmt.Println("\nClosing tunnel")
 	errSession := config.RestAPI.StartSession(config.RestAPI.RefreshToken)
 	errDelete := config.RestAPI.DeleteTunnelAPI(config.Subdomain)
 	if errSession != nil || errDelete != nil {
-		fmt.Fprintf(os.Stderr, "Could not delete tunnel. Use punch cleanup %s", config.Subdomain)
+		fmt.Fprintf(os.Stderr, "Could not delete tunnel. Use punch cleanup %s\n", config.Subdomain)
```

Comment:
"We had some trouble deleting your tunnel.  Use `punch cleanup %s` to make sure we know it's closed."